proxy/envoy/config.go

```diff
@@ -111,6 +115,21 @@ func buildConfig(listeners Listeners, clusters Clusters, mesh *proxyconfig.Proxy
 		},
 	}
 
+	if mesh.StatsdUdpAddress != "" {
```

Comment:
@rshriram the idea is to take advantage of the `statsd_udp_ip_address` field recently added to envoy (in conjunction with the official https://github.com/prometheus/statsd_exporter). This is all to quickly get some (shared) unified visibility into the envoy metrics.